component-interaction/src/app/app-routing.module.ts

```diff
@@ -1,7 +1,26 @@
 import { NgModule } from "@angular/core";
 import { RouterModule, Routes } from "@angular/router";
+import { AppComponent } from "./app.component";
```

Comment:
```suggestion
import { RouterModule, Routes } from "@angular/router";

import { AppComponent } from "./app.component";
```
separate imports from 3rd party dependencies from your custom dependencies